dev/devicelab/bin/tasks/module_custom_host_app_name_test.dart

```diff
@@ -204,7 +202,6 @@ Future<void> main() async {
           <String>['SampleApp:assembleDebug'],
           environment: <String, String>{
             'JAVA_HOME': javaHome,
-            'FLUTTER_ANALYTICS_LOG_FILE': analyticsOutputFile.path,
```

Comment:
Do you know why gradle needed to know about this file?

Reply:
I think this was a mechanism to supress analytics during devicelab tests. Example: https://github.com/flutter/flutter/blob/3391ddf55d36656ea904177301db8e5679b73696/dev/devicelab/bin/tasks/module_test_ios.dart#L472-L474

Code where this is used:

https://github.com/flutter/flutter/blob/a0e43d3053ebb1716d8468b427c6cc44b6f8b7e7/packages/flutter_tools/lib/src/reporting/usage.dart#L152-L153

The question for myself, then, is: was this still necessary? Is this something that should also affect our usage of `unified_analytics`? For example, if I run a devicelab test locally on `master` right now, will I end up sending analytics events?

Reply:
ahh, i'm guessing this is so flutter assemble knows that's it's running as a bot. do we need to preserve this behavior, or will other logic in our bot_detector detect we're in a bot?

Reply:
Oops, sorry, didn't see your reply, sounds like we're on the same page.

Reply:
> if I run a devicelab test locally on `master` right now, will I end up sending analytics events?

Indeed, this appears to be the case. I'll file a follow-up bug and then patch this PR to fix that as well.

Reply:
Filed https://github.com/flutter/flutter/issues/151560

Reply:
SGTM